databases/week3/tables.sql

```diff
@@ -0,0 +1,39 @@
+CREATE DATABASE meal_sharing_db;
+
+USE meal_sharing_db;
+
+-- Meal table
+CREATE TABLE meal(
+id INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+title VARCHAR(255) NOT NULL,
+description TEXT NULL DEFAULT NULL,
+location VARCHAR(255) NOT NULL,   
+_when DATETIME NOT NULL,
```

Comment:
I see that you encountered problems with calling the column "when". The reason for that is that it is a reserved word in mysql: https://dev.mysql.com/doc/refman/8.0/en/keywords.html#keywords-8-0-detailed-W

If you use backticks around the column name it will still be possible to use "when" without an underscore.